core/controllers/editor_test.py

```diff
@@ -524,21 +527,23 @@ def test_exploration_download_handler_for_default_exploration(self):
         response = self.get_custom_response(download_url, 'text/plain')
 
         # Check downloaded zip file.
-        filename = 'oppia-ThetitleforZIPdownloadhandlertest!-v2.zip'
+        filename = 'oppia-b\'ThetitleforZIPdownloadhandlertest!\'-v2.zip'
```

Comment:
This is because of a change I made here: [comment](https://github.com/oppia/oppia/pull/7177/files/848d1530f8758368eb8e85bac1e11465c713c77a#r316121301)

This is required as the request and response headers needs to be in bytes and not unicode. Although I have not yet converted the strings, but this change will be required when I do.

Reply:
But I think this is wrong. The escaping characters shouldn't be making it into the title of the filename. We should be constructing the whole string first and then, if necessary, escaping it (not escaping part of the string and then inserting it into the larger thing).

Reply:
I removed `to_ascii` from `ExplorationFileDownloader` and the 'b' does not come in the output.